x-pack/legacy/plugins/maps/public/actions/map_actions.js

```diff
@@ -452,7 +492,8 @@ export function endDataLoad(layerId, dataId, requestToken, data, meta) {
 }
 
 export function onDataLoadError(layerId, dataId, requestToken, errorMessage) {
-  return async (dispatch) => {
+  return async (dispatch, getState) => {
```

Comment:
Another async that can be removed but just resolve this and leave unchanged if you'd prefer to do that in a different PR

Reply:
added https://github.com/elastic/kibana/issues/37934#issuecomment-529597941 to issue tracking clean-up of actions. Want to avoid making changes in this PR that are outside scope